test/e2e/framework/ssh/ssh.go

```diff
@@ -86,6 +86,11 @@ func GetSigner(provider string) (ssh.Signer, error) {
 		if keyfile == "" {
 			keyfile = "id_rsa"
 		}
+	case "azure":
```

Comment:
@bzsuni this looks unrelated to the rest of PR changes. Would it make sense to submit it separately?

Reply:
There is a problem with my commit, I will make some changes later

Reply:
I have made the necessary modifications, thanks @bart0sh